apps/namadillo/src/App/Staking/ReDelegate.tsx

```diff
@@ -41,7 +41,7 @@ export const ReDelegate = (): JSX.Element => {
     updatedAmountByAddress: amountsRemovedByAddress,
     onChangeValidatorAmount,
     myValidators,
-  } = useStakeModule({ account });
+  } = useStakeModule({ account: account });
```

Comment:
Is this change from `{ account }` to `{ account: account }` intended?

Reply:
:O can't believe I left this one behind. Thanks!